examples/echo_app.rs

```diff
@@ -0,0 +1,52 @@
+extern crate abci;
+use abci::*;
+
+struct EchoApp;
+
+impl Application for EchoApp {
```

Comment:
Does not look like an echo app to me) more like nilapp or emptyapp

Reply:
You're right. EmptyApp sounds better.